From 1cb83759632bb218559c4d0d65ff79f868c03dc5 Mon Sep 17 00:00:00 2001
From: "Maxin B. John" <maxin.john@intel.com>
Date: Tue, 9 Aug 2016 17:03:31 +0300
Subject: [PATCH] add geometry input when calibrating

Send monitor geometry to xinput_calibrator when running the script
Update:
        Remove bashism

Upstream-Status: Inappropriate [no longer maintained]

Signed-off-by: Jonathan David <jonathan.david@ni.com>
Signed-off-by: Maxin B. John <maxin.john@intel.com>
---
 scripts/xinput_calibrator_pointercal.sh | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/scripts/xinput_calibrator_pointercal.sh b/scripts/xinput_calibrator_pointercal.sh
index fccb197..fea7c2f 100755
--- a/scripts/xinput_calibrator_pointercal.sh
+++ b/scripts/xinput_calibrator_pointercal.sh
@@ -24,7 +24,8 @@ if [ -e $CALFILE ] ; then
   fi
 fi
 
-CALDATA=`$BINARY --output-type xinput -v | tee $LOGFILE | grep '    xinput set' | sed 's/^    //g; s/$/;/g'`
+RESOLUTION=$(xrandr | awk -F '[[:space:]+]' '/ connected/ { if ($3 != "primary") print $3; if ($3 == "primary") print $4 }')
+CALDATA=`$BINARY --geometry $RESOLUTION --output-type xinput -v | tee $LOGFILE | grep '    xinput set' | sed 's/^    //g; s/$/;/g'`
 if [ ! -z "$CALDATA" ] ; then
   echo $CALDATA > $CALFILE
   echo "Calibration data stored in $CALFILE (log in $LOGFILE)"
-- 
2.4.0

